frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -464,6 +464,15 @@ search-FiltersPanel--heading-status = Translation Status
 search-FiltersPanel--heading-tags = Tags
 search-FiltersPanel--heading-extra = Extra Filters
 search-FiltersPanel--heading-authors = Translation Authors
+search-FiltersPanel--status-name-all = All
+search-FiltersPanel--status-name-translated = Translated
+search-FiltersPanel--status-name-fuzzy = Fuzzy
+search-FiltersPanel--status-name-warnings = Warnings
+search-FiltersPanel--status-name-errors = Errors
+search-FiltersPanel--status-name-missing = Missing
+search-FiltersPanel--status-name-unreviewed = Unreviewed
+search-FiltersPanel--extra-name-unchanged = Unchanged
+search-FiltersPanel--extra-name-rejected = Rejected
```

Comment:
You're missing the newly added Empty Filter here.

Reply:
Yeah. Will add that also